UPGRADING

```diff
@@ -50,6 +50,18 @@ PHP 8.5 UPGRADE NOTES
   . pcntl_exec() now throws ValueErrors when entries or keys of the
     $env_vars parameter contain null bytes.
 
+- PDO:
+  . The constructor arguments set in conjunction with PDO::FETCH_CLASS now
+    follow the usual CUFA (call_user_func_array) semantics.
+    This means string keys will act like a named argument.
+    Moreover, automatic wrapping for by-value arguments passed to a by-ref
+    parameter has been removed, and the usual E_WARNING about this is now
+    emitted.
+    To pass a variable by-ref to a constructor argument use the general
+    array value reference assignment: $ctor_args = [&$valByRef]
+  . Attempting to modify a PDOStatement during a call to PDO::fetch(),
```

Comment:
Could this be worded better? I assume you mean that the default fetch mode cannot be changed, right? It seems like I can still make other changes to the object, e.g. execute it again. Also, it's not immediately clear what is meant by "during a call".

Reposting the comment for better visibility.

Reply:
Also, while it's now forbidden to change the default fetch mode, the actual fetch mode can still be changed via the parameter to `fetch`/`fetchAll`. 

I wonder if there exists a driver that allows the default fetch mode to be changed via `PDOStatement::setAttribute`.

What exactly is the reason behind introduction of this error?